src/utils/build-url.ts

```diff
@@ -1,7 +1,12 @@
+import { URL } from 'url';
+
 export function buildUrl(
     url: string,
     serviceName: string,
     serviceVersion: string
 ) {
-    return `${url}/contracts/services/${serviceName}/versions/${serviceVersion}`;
+    return new URL(
+        `/contracts/services/${serviceName}/versions/${serviceVersion}`,
+        url
+    ).toString();
```

Comment:
First arguments needs to be a relative URL (starting with `./`). Otherwise this will not work with judge-d URL exposed on non-root paths:

```
new URL('/contracts/services/app/versions/123', 'http://example.com/some-path/').toString()
// => "http://example.com/contracts/services/app/versions/123"
```

see https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
Also for this reason, it would be worth adding a unit test covering cases of `url`

Reply:
Fixed and tests added.